src/components/common/Footer.tsx

```diff
@@ -25,7 +26,16 @@ const Footer: React.FC = () => (
             // TODO: 仮リンクではなく、別ファイルで定義したものを参照する
           }
           <a href="/">
-            <li>プライバシーポリシー</li>
+            <li className="pb-4">プライバシーポリシー</li>
+          </a>
+          <a href="https://www.gunma-ct.ac.jp/">
+            <li className="pb-4">群馬高専公式サイト</li>
+          </a>
+          <a href={Static.instagram.url}>
+            <li className="pb-4">工華祭公式Instagram</li>
```

Comment:
```suggestion
            <li className="pb-4">Instagram</li>
```

工華祭公式 がなくても伝わる気がしますし、スマートだと思います